docs/deploy-full-rp-service-in-dev.md

```diff
@@ -20,6 +20,36 @@
       to the tag expected by aro deployer
     - with a dirty tag, it's not clear what's actually in the image
 
+## Containerize and Automate the int-like Development RP
+
+The [manual int-like Development RP deployment](#deploying-an-int-like-development-rp) is automated using scripts and functions that ease the process, and reduce the setup toll.
+Run the below command to automate the full RP int-like dev env using a container:
+
+   ```bash
+    AZURE_PREFIX=zzz RP_LOCATION=eastus SKIP_DEPLOYMENTS=true make full-rp-dev
+   ```
+
+- Prior to running the automation a user must be logged in to Azure locally (with Azure CLI), since the local Azure credentials (from `${HOME}/.azure`) are copied to the container.
+- Running the automation will be skipped in case the RP and GWY VMSSs succeed and their final deployment succeeds as well.
+- **AZURE_PREFIX** must be unique to avoid Azure resources collision between other deveolopers.
+ This value is also used for truncating the keyVault prefix up to 20 characters.
+- (optional) Set **RP_LOCATION** var to your preferred the Azure location (by default it is 'eastus').
+- (optional) Set **SKIP_DEPLOYMENTS** var as false (by default 'true') when you prefer to deploy Azure resources regardless of their existence.
+- (optional) You may run the target with *RP_FULL_DEV_IMAGE* var (e.g., `RP_FULL_DEV_IMAGE=YOUR_REPO_AND_TAG`) in case you want to push the container to your registry.
+- For running locally the RP helper functions please use *usage_full_rp_funcs* and *usage_rp_dev* to get functions' usage help for [*full_rp_funcs.sh*](https://github.com/Azure/ARO-RP/blob/master/hack/rp-dev/full_rp_funcs.sh) and [*rp_dev_helper.sh*](https://github.com/Azure/ARO-RP/blob/master/hack/devtools/rp_dev_helper.sh) respectively
+
+- Clean Azure resources (4 ResourceGroups and 4 KeyVaults) by running one of the following (based on the above full RP creation):
+
+   ```bash
+   AZURE_PREFIX=zzz clean_rp_dev_env eastus
+   ```
+
+   Or
+
+   ```bash
+   AZURE_PREFIX=zzz RP_LOCATION=eastus make full-rp-dev-clenup
+   ```
+
 ## Deploying an int-like Development RP
```

Comment:
If this is the manual approach, would be good to make it clear the difference between what is the Containerized approach and Manual one in the title header, by adding something like "(Manual)"

Reply:
Good point 👍🏻